LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2019 Oleksii Dykan
```

Comment:
Coincidentally we opened #17 right at the same time with `wearereasonablepeople` as the copyright holder and it got me thinking.

I have a suspicion that technically, wearereasonablepeople *is* the appropriate entity to hold the copyright, because of a clause in Oleksii's contract that was active while he produced this work. 

I'm fairly sure though, that whoever ended up owning the copyrights "officially" would be fully willing to transfer that to right to Oleksii. I think it might be @chris-warp as the "employer", but I'm not entirely sure about this either.

I will ask around next week to ensure this LICENSE can be considered valid.

Reply:
That sounds great. I have no clue on the back history of the repo.

Reply:
Our view is that the copyright is currently formally held by Oleksii's employee at the time, WEAREREASONABLEPEOPLE. As such, I feel more comfortable merging this if the copyright holder in the license is changed accordingly, like we have in our [other licenses](https://github.com/wearereasonablepeople/warped-reducers/blob/245798a9df3f788658fdce9ced1231ba88fc0b2c/LICENSE#L1-L2), if only to ensure that the license is as valid as possible.

I believe copyright *can* be formally transferred to another party, such as @alickbass, by a representative of the WEAREREASONABLEPEOPLE company, which I believe I am not. But in the interest of closing #14, I think it'll be faster to look into that after merging this with WEAREREASONABLEPEOPLE as the copyright holder.